fastlane/lib/fastlane/setup/setup.rb

```diff
@@ -27,6 +42,7 @@ def run(user: nil, is_swift_fastfile: false)
 
       if platform == :ios
         SetupIos.new.run(user: user, is_swift_fastfile: is_swift_fastfile)
+        Fastlane::SwiftLaneManager.first_time_setup if is_swift_fastfile        
```

Comment:
This should maybe be moved even deeper (into the `SetupIos` class)

Reply:
I agree, and it could run at the end of `SetupIos.new.run`